OKR.md

```diff
@@ -2,6 +2,27 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q4
+
+**Debuggability and Stability**
+- error handling is done correctly and error codes are part of the API - @jacobheun
+- libp2p has a network visualization tool - @vasco-santos
+- js-libp2p has a daemon with a RPC API and a client library - @jacobheun
+- interop tests for go-libp2p and js-libp2p exist for core functionality - @jacobheun
+- a testbed exists and can test 50,000 libp2p nodes (browser and node) - @jacobheun
+- the testbed can run large scale tests for the DHT and Pubsub - @vasco-santos
+
+**Connectivity Magic is better and more reliable**
+- the DHT is enabled by default in js-ipfs and is interoperable with go - @vasco-santos
+- IPNS works over DHT and is interoperable with go - @vasco-santos
+- IPNS spec exists and is complete - @vasco-santos
+- Nat traversal Spec is ready for js-libp2p to implement - @jacobheun
+
+**Libp2p is seen as the go to p2p networking stack**
+- usage examples exist for the key modules of libp2p - @jacobheun
```

Comment:
I agree! Even when I started creating a new suite of tests for DHT interop, I tried to find examples both on go and js, but there was any. However, I would like something more. 

I think we should aim to have examples for the key modules, as well as have a general overview to the `README` of several modules, which are really incomplete. Something like `Usage examples and proper documentation exist for the key modules of libp2p`. What do you think?

Reply:
Yes, I agree. Many of the recent questions that have come up could be answered by better/clearer answers to `what the modules does` and `how configuration options affect that`